soak/README.md

```diff
@@ -301,35 +274,33 @@ Validation:
 * After executing above command, a Kubernetes Job will start which will execute the soak tests until complete. 
 * Run `kubectl get job/$SERVICE_NAME-soak-test`  and validate that the resource exists.
 
-### Step 7 (Monitor the metrics using Grafana console)
+### Step 6 (Monitor the metrics using Grafana console)
 
 * As mentioned in step 2, Prometheus and Grafana services are exposed through background port-forward processes.
 * You can access the Grafana dashboard in your browser at `http://127.0.0.1:$LOCAL_GRAFANA_PORT` address. NOTE: Do not forget
 to substitute "$LOCAL_GRAFANA_PORT" with actual value from your `~/.ack/soak/config` file
 * When Grafana console is loaded, you can login with default username `admin` and default password `prom-operator`
 NOTE: You can see the value of these credentials in secret named `$PROM_CHART_RELEASE_NAME-grafana`
 * After the login, you can view your Grafana dashboard at `http://127.0.0.1:3000/dashboards`.
+* The `ACK Dashboard` will show the ACK-specific request counts and error codes when making calls to AWS APIs.
 * `Kubernetes/Compute Resources/Pod` dashboard will show the resource consumption by the controller pod
-* The dashboard imported in step 4 will show the ACK dashboard with request counts and error codes when making calls to AWS APIs.
 
 
-### Step 8 (Cleanup Soak test chart)
+### Step 7 (Cleanup Soak test chart)
 
 * RUN `helm uninstall $SOAK_RUNNER_CHART_RELEASE_NAME`
 
 
-### Step 9 (Cleanup the service controller deployment)
+### Step 8 (Cleanup the service controller deployment)
```

Comment:
Since the instructions now create new namespaces, please add deleting namespaces into cleanup instructions. (Helm will not delete namespaces on uninstall command)